src/parser/parser.ee.ts

```diff
@@ -23,10 +24,25 @@ export async function isFileSupported({
   }
 
   // OOXML
-  // eslint-disable-next-line sonarjs/prefer-single-boolean-return
   if (filename.endsWith('.docx')) {
     return true;
   }
+  if (filename.endsWith('xlsx')) {
+    return true;
+  }
+  if (filename.endsWith('.odt')) {
```

Comment:
libre office format =/= OOXML

Separate them please, to make it easier to maintain


so:
```ts
// OOXML
if ( ... )
/* your MS Office formats */

// Open Office XML
if ( ... )
/* your libre office / open office formats */
```